1/ToDoApp/package.json

```
@@ -7,7 +7,9 @@
   "private": true,
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint src"
```

```suggestion
    "lint": "eslint src"
```
Both npm and yarn will look into `./node_modules/.bin/` to resolve a binary. That's the reason to run `npm run` or `yarn run`: You don't have to install binaries globally.